docs/cloud/pouta/multiattach.md

```diff
@@ -0,0 +1,429 @@
+
+$ ansible-playbook main.yml -e fs='gfs2'
+
+$ for i in $(seq 1 16);
+do
+    openstack --os-compute-api-version 2.60 server add volume "cinder-gfs2-$i" multi-attach-test-gfs2
+done
+
+$ ansible-playbook main.yml -e fs='gfs2'
+```
+
+You need to run Ansible twice due to a bug in the `openstack.cloud.server_volume` which can only attach the volume to a single VM and fails with the other ones.
+
+If you already have a cluster of VMs, or want to manually create them, it is still possible to use the `gfs2` Ansible role. The steps are simple:
+
+1. Create and attach the volume. See the manual [Create and attach a volume](#create-and-attach-a-volume) from above.
+
+1. Create a standard Ansible inventory like this one:
+
+    ```ini
+    [all]
+    <VM_name> ansible_host=192.168.1.XXX ansible_user=<user>
+    # ...
+    [all:vars]
+    ansible_ssh_common_args='-J <jumphost>'
+    ```
+
+    In the example above you need to replace `<VM_name>` by the name of the VM, the IP `192.168.1.XXX` must be the correct IP of the VM, and finally the `<user>` has to also be replaced by the corresponding one. You need to have a line per VM node that you want to include in the cluster. Finally, if you are using a Jump Host, you need to replace `<jumphost>` by its connection information, like `ubuntu@177.51.170.99` 
+
+1. Create a playbook like this one:
+
+    ```yaml
+    ---
+
+    - name: Configure VMs
+      hosts: all
+      gather_facts: true
+      become: true
+      roles:
+        - role: hosts
+        - role: gfs2
+    ```
+
+    This will run two roles, the `hosts` one if to create a `/etc/hosts` file in every VM with the IPs and names of every VM. The `gfs2` role installs and configures the cluster.
+
+1. And run it:
+
+    ```sh
+    $ ansible-playbook main-gfs2.yml -i inventory.ini
+    ```
+
+### GFS2 manual install
+
+In order to install GFS2, you need to follow few steps:
+
+1. Install the VM nodes. There is no special consideration on this step, other than making sure the nodes can see each other in the Network (it is the default behaviour of VM nodes created in the same Pouta project), and that they are installed with the same distribution version. We have tested this with `Ubuntu v22.04` and `AlmaLinux-9`, other distributions and versions might also work, but we have not tested them.
+
+1. Create and attach the volume. See the manual [Create and attach a volume](#create-and-attach-a-volume) from above.
+
+1. Install the GFS2 software. This step is distribution dependent.
+
+    1. For AlmaLinux and other RedHat based distributions you just need enable two collections and install few packages:
+
+    ```sh
+    sudo dnf config-manager --enable  highavailability resilientstorage
+    sudo dnf install pacemaker corosync pcs dlm gfs2-utils
+    ```
+
+    1. For Ubuntu and other Debian based distributions you just need to install 3 packages:
+
+    ```sh
+    apt install gfs2-utils dlm-controld linux-modules-extra-<ansible_kernel>
```

Comment:
I don't really understand the meaning of `ansible` in `<ansible_kernel>`. Did you mean ubuntu kernel? If so, maybe this command could replace:
```sh
apt install gfs2-utils dlm-controld linux-modules-extra-$(uname -r)
```